Asset/src/V1beta1/Gapic/AssetServiceGapicClient.php

```diff
@@ -0,0 +1,489 @@
+     *           See the {@see \Google\ApiCore\Transport\GrpcTransport::build()} and
+     *           {@see \Google\ApiCore\Transport\RestTransport::build()} methods for the
+     *           supported options.
+     * }
+     *
+     * @throws ValidationException
+     * @experimental
+     */
+    public function __construct(array $options = [])
+    {
+        $clientOptions = $this->buildClientOptions($options);
+        $this->setClientOptions($clientOptions);
+        $this->operationsClient = $this->createOperationsClient($clientOptions);
+    }
+
+    /**
+     * Exports assets with time and resource types to a given Google Cloud Storage
+     * location. The output format is newline delimited JSON.
+     * This API implements the [google.longrunning.Operation][google.longrunning.Operation] API allowing users
+     * to keep track of the export.
+     *
+     * Sample code:
+     * ```
+     * $assetServiceClient = new AssetServiceClient();
+     * try {
+     *     $formattedParent = $assetServiceClient->projectName('[PROJECT]');
+     *     $contentTypes = [];
+     *     $outputConfig = new OutputConfig();
+     *     $operationResponse = $assetServiceClient->exportAssets($formattedParent, $contentTypes, $outputConfig);
+     *     $operationResponse->pollUntilComplete();
+     *     if ($operationResponse->operationSucceeded()) {
+     *         $result = $operationResponse->getResult();
+     *         // doSomethingWith($result)
+     *     } else {
+     *         $error = $operationResponse->getError();
+     *         // handleError($error)
+     *     }
+     *
+     *
+     *     // Alternatively:
+     *
+     *     // start the operation, keep the operation name, and resume later
+     *     $operationResponse = $assetServiceClient->exportAssets($formattedParent, $contentTypes, $outputConfig);
+     *     $operationName = $operationResponse->getName();
+     *     // ... do other work
+     *     $newOperationResponse = $assetServiceClient->resumeOperation($operationName, 'exportAssets');
+     *     while (!$newOperationResponse->isDone()) {
+     *         // ... do other work
+     *         $newOperationResponse->reload();
+     *     }
+     *     if ($newOperationResponse->operationSucceeded()) {
+     *       $result = $newOperationResponse->getResult();
+     *       // doSomethingWith($result)
+     *     } else {
+     *       $error = $newOperationResponse->getError();
+     *       // handleError($error)
+     *     }
+     * } finally {
+     *     $assetServiceClient->close();
+     * }
+     * ```
+     *
+     * @param string       $parent       Required. The relative name of the root asset. It can only be an
+     *                                   organization number (e.g. "organizations/123") or a project number
+     *                                   (e.g. "projects/12345").
+     * @param int[]        $contentTypes A list of asset content types. If specified, only matching content will be
```

Comment:
Does it seem worthwhile to see if this should be marked as optional in the gapic config?

Reply:
That's a good point. I'll ask the product eng